### PR TITLE
Various fixes

### PR DESCRIPTION
**Pull request by knikolla**
* Calls to glance now should be done in /image and calls to
  cinder should be done in /volume. So the endpoint should be
  registered like that.
* Fixed the aggregator breaking streaming.
* Always stream on GET and PUT.

### CODE REVIEW

File: mixmatch/proxy.py [hunk 1]
```diff
@@ -23,62 +25,55 @@
...
-GLANCE_APIS = ['images', 'schemas', 'metadefs', 'tasks']
-
 
 def stream_response(response):
     yield response.raw.read()
 
 
+def is_valid_uuid(value):
+    try:
+        _ = uuid.UUID(value, version=4)  # noqa
```
Comment (gsilvis) [inline mixmatch/proxy.py]: Why not just ``uuid.UUID(value, version=4)`` as the whole line?  would that fix the pep8 issue?

Comment (knikolla) [inline mixmatch/proxy.py]: Just realized. Will update.

**Comment by gsilvis** [comment]
Can you squash this?

**Comment by knikolla** [comment]
Sure.

**Comment by gsilvis** [comment]
+1

**Pull request merged**